.github/workflows/prlabelling.yaml

```diff
@@ -0,0 +1,16 @@
+name: "Pull Request Labeler"
+run-name: ${{ github.actor }} created Pull Request Labeler
+
+on:
+- pull_request_target
+
+jobs:
```

Comment:
It would be great if you could filter for PRs created by @dependabot
To do so you can follow this example [here](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#accessing-and-using-event-properties)

Reply:
should i assign permissions ,contents and pull request or just leave it for default

Reply:
I think this looks fine for now